winstone/src/javax/servlet/http/HttpServlet.java

```diff
@@ -103,14 +102,14 @@ protected void service(HttpServletRequest request,
             if (lastModified == -1)
                 doGet(request, response);
             else {
-                long ifModifiedSince = request.getDateHeader(HEADER_IFMODSINCE);
-                if (ifModifiedSince < (lastModified / 1000 * 1000)) {
-                    if (!response.containsHeader(HEADER_LASTMOD)
-                            && (lastModified >= 0))
-                        response.setDateHeader(HEADER_LASTMOD, lastModified);
+//                long ifModifiedSince = request.getDateHeader(HEADER_IFMODSINCE);
```

Comment:
Could you add a comment here that links to https://github.com/cubing/tnoodle/issues/215?